openlibrary/plugins/upstream/models.py

```diff
@@ -5,6 +5,7 @@
 import simplejson
 import re
 from collections import defaultdict
+from isbnlib import canonical
```

Comment:
I'm kinda hoping this is unnecessary,  `openlibrary.utils.isbn` should already be returning canonical formed isbns.